lib/mapping/CMap.h

```diff
@@ -260,7 +260,7 @@ enum class EMapFormat: uint8_t
 	SOD   = 0x1c, // 28

```

Comment:
Only as information, has nothing to do with the pull request: There is another official format.

For the Chronicles maps and campaigns. But I think this was mainly to prevent the Chronicles campaigns from being loaded into SoD. I don't think there is much difference.

https://github.com/potmdehex/homm3tools/blob/master/h3m/h3mlib/h3m_constants/h3m_format.h

https://github.com/potmdehex/homm3tools/blob/master/h3c/h3clib/h3clib.h

Reply:
Will add note in our source code, might be useful info in the future, once we can properly support datasets other than sod